packages/AsyncLoader/package.json

```diff
@@ -5,27 +5,14 @@
   "main": "AsyncLoaderElement.js",
   "scripts": {
     "test": "npx ava --verbose",
-    "build": "npx rollup --c rollup.config.mjs"
+    "build": "npx rollup --c rollup.config.mjs",
+    "lint": "npx eslint \"src/**/*.{js,mjs}\" -c ../../node_modules/@joinbox/eslint-config-joinbox/index.js"
   },
   "author": "Felix Steiner <felix@joinbox.com>",
   "license": "ISC",
-  "ava": {
-    "files": [
-      "**/*.unit.mjs",
-      "**/*.integration.mjs"
-    ],
-    "nodeArguments": [
-      "--experimental-modules"
-    ]
-  },
-  "devDependencies": {
-    "@babel/eslint-parser": "^7.22.5",
-    "@joinbox/eslint-config-joinbox": "^1.0.11",
-    "@rollup/plugin-terser": "^0.4.4",
-    "ava": "^5.3.1",
-    "jsdom": "^22.1.0",
-    "lerna": "^7.1.0",
-    "rollup": "^3.25.3"
+  "type": "module",
+  "dependencies": {
+    "@joinbox/ui-tools": "*"
```

Comment:
Müssen wir hier die `devDependencies` gar nicht mehr erfassen? Wenn jemand das Paket installiert in einem nicht-produktiv-Environment und die Tests ausführen möchte, würden die entsprechenden Packages fehlen, oder?

Reply:
Müssen wir auch noch besprechen.